index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Notes App</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div class="main">
+        <div class="sidebar">
+            <label for="dark">
+                <input type="checkbox" name="dark" id="dark" />
+                <span class="slider"></span>
+            </label>
+            <button style="margin-top: 1rem;" id="btn-new" class="btn btn-new">+ New Note</button>
+        </div>
+        <div class="note">
+            <div class="absolute">
+                <button id="btn-save" class="btn btn-new btn-save">Save</button>
+                <button id="btn-delete" class="btn btn-delete">Delete</button>
+            </div>
+            <input type="text" id="heading" class="note-heading" />
+            <div class="note-content" id="content" contenteditable="true"></div>
```

Comment:
```suggestion
            <div class="note-content" id="content" contenteditable></div>
```
anytime you're setting a `true` on an attribute, it's okay to not include true explicitly.

Reply:
Yup. Small things that make code look cleaner. :)